src/x86-64.h

```diff
@@ -74,11 +76,13 @@
 #define JOHN_AVX			1
 #endif
 
-#ifdef JOHN_AVX512BW
+#if defined(JOHN_AVX512BW) || defined(JOHN_AVX512VL) || defined(JOHN_AVX512DQ)
 #define CPU_DETECT			1
 #define CPU_REQ				1
 #define CPU_REQ_AVX512BW		1
-#define CPU_NAME			"AVX512BW"
+#define CPU_REQ_AVX512VL		1
+#define CPU_REQ_AVX512DQ		1
+#define CPU_NAME			"AVX512(BW+VL+DQ)"
```

Comment:
We can keep all 3 mentioned in `CPU_NAME`, for reporting in the "Sorry" line. (No further change is needed here.)

Reply:
The addition of `CPU_REQ_AVX512VL` (+DQ) is also required.